rasa/shared/utils/cli.py

```diff
@@ -1,10 +1,28 @@
+import fcntl
+import os
 import sys
 from typing import Any, Text, NoReturn
 
 import rasa.shared.utils.io
 
 
+def print_blocking(string) -> None:
+    """Save fcntl settings and restore them after print()."""
```

Comment:
I'm unfamiliar with `fcntl`, could we expand the docstring a bit more about the need for this.